spyderlib/config.py

```diff
@@ -530,7 +560,9 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Meta+S',
               'editor/print': "Ctrl+P",
+              'editor/close file': 'Ctrl+W',

```

Comment:
I removed this one in 2.3.3 to make it a fixed shortcut. Why are you adding it again?